config/helm/chart/default/tests/Common/edgeconnect/serviceaccount-edgeconnect_test.yaml

```diff
@@ -18,4 +18,4 @@ tests:
       rbac.edgeConnect.create: false
     asserts:
       - hasDocuments:
-        count: 0
+          count: 0
```

Comment:
I think this change is redundant.

Reply:
try it with `count: 1` and check if it fails

Reply:
my point was, why do we want to change format of all files?

Reply:
Because it's currently wrong and only works by chance. 🙂

@andriisoldatenko You can verify that locally by changing the value of `count` to something other than `0`, running the unit tests and seeing that they still succeed. If you want to not do these changes in this PR feel free to create a separate PR with these fixes - I'm fine with that as well.

Reply:
nah i'm fine, just a bit surprised about yaml :)